assets/elm/DrinksCarousel.elm

```diff
@@ -0,0 +1,192 @@
+port module DrinksCarousel exposing (..)
+
+import Browser
+import Http
+import Json.Decode as Decode exposing (..)
+import Json.Decode.Pipeline exposing (..)
+import Html exposing (..)
+import Html.Events exposing (..)
+import Html.Attributes exposing (..)
+import Html.Events exposing (onClick)
+import Array exposing (..)
+
+
+-- MAIN
+
+
+main =
+    Browser.element
+        { init = init
+        , subscriptions = subscriptions
+        , update = update
+        , view = view
+        }
+
+
+
+-- MODEL
+
+
+getDrinks : Cmd Msg
+getDrinks =
+    Http.get ("/json_drinks") drinksDecoder |> Http.send ReceiveDrinks
+
+
+drinksDecoder : Decoder (List Drink)
+drinksDecoder =
+    Decode.list drinkDecoder
+
+
+drinkDecoder : Decoder Drink
+drinkDecoder =
+    Decode.map3 Drink
```

Comment:
consider using decode pipeline in next iteration. https://github.com/NoRedInk/elm-json-decode-pipeline 👍